style.css

```diff
@@ -0,0 +1,727 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
Please do not disable linter rules instead fix all the errors spotted by the linter.